app/src/main/java/com/github/se/eventradar/ui/theme/Color.kt

```diff
@@ -2,10 +2,57 @@ package com.github.se.eventradar.ui.theme
 
 import androidx.compose.ui.graphics.Color
 
-val Purple80 = Color(0xFFD0BCFF)
-val PurpleGrey80 = Color(0xFFCCC2DC)
-val Pink80 = Color(0xFFEFB8C8)
+val P_10 = Color(0xFF21005D)
```

Comment:
Code documentation/style wise, might be better to name these what the colour is, rather than just P_ e.g. Purple_10

Reply:
I agree, except that P do not stand for "purple" but for "primary". In the reference it is named like this and I did the same because looking up for each individual color name would be too time consuming. And it was also easier to then assign the color to the material

Reply:
Makes sense, thanks for the explanation :)